tests/queries/0_stateless/00662_has_nullable.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS 00662_has_nullable;
+CREATE TABLE 00662_has_nullable(a Nullable(UInt64)) ENGINE = Memory;
+
+INSERT INTO 00662_has_nullable VALUES (1), (Null);
+SELECT a, has([0, 1], a) FROM 00662_has_nullable;
+
+DROP TABLE 00662_has_nullable;
```

Comment:
Would be also good to add more tests cases (more values in nullable table, put nullable arguments as left/both arguments in `has()` function.

Reply:
Will do

Reply:
@yariks5s also JFYI the cases you described actually exist in https://github.com/ClickHouse/ClickHouse/blob/master/tests/queries/0_stateless/00662_array_has_nullable.sql

I was thinking about expanding it instead of writing the new file, but that test didn't have any table creation, hence I decided to create a new one.